tools/src/test/java/org/apache/kafka/tools/consumer/group/ConsumerGroupCommandTestUtils.java

```diff
@@ -85,6 +88,9 @@ static List<ClusterConfig> forKRaftGroupCoordinator() {
         Map<String, String> serverProperties = new HashMap<>();
         serverProperties.put(OFFSETS_TOPIC_PARTITIONS_CONFIG, "1");
         serverProperties.put(OFFSETS_TOPIC_REPLICATION_FACTOR_CONFIG, "1");
+        serverProperties.put(GROUP_INITIAL_REBALANCE_DELAY_MS_CONFIG, "1000");
```

Comment:
Hi @m1a2st, one question, based on the document of `group.initial.rebalance.delay.ms`. ([Link](https://kafka.apache.org/documentation/#consumerconfigs_max.poll.interval.ms:~:text=A%20new%20config,up%20to%20a))

> The default value for this is 3 seconds. During development and testing it might be desirable to set this to 0 in order to not delay test execution time.

I think setting `group.initial.rebalance.delay.ms` to zero might be better. Is there have any reason we use 1000 ms?
I can see some tests are using 1000ms, but some others are using 0ms.

Reply:
Hi @chenyulin0719, You can see this [discussion](https://github.com/apache/kafka/pull/16810#issuecomment-2275568303), we set the `group.initial.rebalance.delay.ms` from 3000 to 1000  to not only keep the initial rebalance delay but also speed up the test.

Reply:
I see. Comparing to set `group.initial.rebalance.delay.ms` to 0 ms,  adding some delay could speed up the metadata synchronization in Kraft mode. Let's a suprising finding to me.

Thanks for sharing the discussion.

Reply:
>  I think setting group.initial.rebalance.delay.ms to zero might be better.

That is good, but we want to have delay in testing. That can "reduce" the difference between test and production. Also, 1.5s is good enough in speedup.